src/basic/def.h

```diff
@@ -64,4 +64,5 @@
                 .un.sun_path = "\0/org/freedesktop/plymouthd",  \
         }
 
-#define VARLINK_ADDR_PATH_MANAGED_OOM "/run/systemd/io.system.ManagedOOM"
+#define VARLINK_ADDR_PATH_MANAGED_OOM_SYSTEM "/run/systemd/io.system.ManagedOOM"
```

Comment:
maybe add comment here, that this is what PID 1 listens on, and where oomd is a client.